osi_detectedobject.proto

```diff
@@ -84,6 +84,27 @@ message DetectedItemHeader
         // Values provided by tracking only.
         //
         MEASUREMENT_STATE_PREDICTED = 3;
+
+        // This indicates the first occurrence of this particular object in the object list.
+        //
+        MEASUREMENT_STATE_NEW = 4;
+
+        // The object was partly updated with new information
+        // from a sensor in the last update cycle of this object
+        // list. The measurement was incomplete and could,
+        // therefore, only partially update the signals of this
+        // object. The remaining signals of this object are unchanged
+        // or predicted for this measurement cycle.
+        //
+        MEASUREMENT_STATE_PARTIALLY_MEASURED = 5;
```

Comment:
PARTLY_MEASURED in ISO23150?

Reply:
PARTLY_MEASURED is correct. This is a mistake on ADI side.